cursorless-talon/src/cheatsheet_html/cheat_sheet.py

```diff
@@ -12,12 +12,29 @@
 from .sections.special_marks import get_special_marks
 
 mod = Module()
+ctx = Context()
+ctx.matches = r"""
+app: vscode
+"""
 
 cheatsheet_out_dir = Path(tempfile.mkdtemp())
 instructions_url = "https://www.cursorless.org/docs/"
 
 
 @mod.action_class
+class Actions:
+    def cursorless_cheat_sheet_show_html():
+        """Show new cursorless html cheat sheet"""
+        app.notify(
+            'Please first focus an app that supports cursorless, eg say "focus code"'
+        )
+
+    def cursorless_open_instructions():
+        """Open web page with cursorless instructions"""
+        webbrowser.open(instructions_url)
```

Comment:
no reason not to support this everywhere.  This way if user first installs Cursorless and says "cursorless help", they'll get the docs no matter what